test/clj/cider/nrepl/middleware/out_test.clj

```diff
@@ -102,3 +102,20 @@
           (.write printer (char-array " bye67") 1 3)
           (is (= "bye" (.toString out-writer)))
           (is (= "bye" (.toString message-writer))))))))
+
+(defn find-thread [thread-name]
+  (->> (Thread/getAllStackTraces)
+       (.keySet)
+       (filter #(= thread-name (.getName ^Thread %)))
+       first))
+
+(deftest print-stream-flush-test
+  (let [out-writer (StringWriter.)
+        printer (o/print-stream (volatile! out-writer))]
+    (.write printer 32)
```

Comment:
`\space` perhaps?

Reply:
Actually, `\space` is a char and not an int.

Reply:
One can do `(int \space)` though 😇

Thanks for applying all the feedback regardless